### ISSUE TITLE
tracker_run_name and tracker_project_name not working for wandb

### ISSUE THREAD

**Issue by zhuliyi0** [bghira/SimpleTuner#91]
wandb shows tracker_run_name as project name, and random run names.

I ran this command directly in console:

accelerate launch --mixed_precision="bf16" \
	--num_processes="1" \
	--num_machines="1" \
	--dynamo_backend="no" \
	 train_sdxl.py \
	--pretrained_model_name_or_path="stabilityai/stable-diffusion-xl-base-1.0" \
	--num_train_epochs="10" \
	--learning_rate="8e-7" \
	--lr_scheduler="constant" \
	--seed=1234 \
	--random_flip \
	--instance_data_dir="/root/autodl-tmp/Dataset/lvchengguiyuxi" \
	--seen_state_path="/root/autodl-tmp/simpletunerOutput/lvchengguiyuxi/training_images_seen.json" \
	--state_path="/root/autodl-tmp/simpletunerOutput/lvchengguiyuxi/training_state.json" \
	--output_dir="/root/autodl-tmp/simpletunerOutput/lvchengguiyuxi" \
	--train_batch="1" \
	--gradient_accumulation_steps="1" \
	--caption_dropout_probability=0.05 \
	--validation_prompt="a photo of  lower part of a lvchenguiyuxi style high-rise building, walkway, trees, bush, background building, stone sculpture on the ground" \
	--num_validation_images=1 \
	--resolution="768" \
	--validation_resolution="1024" \
	--checkpointing_steps="800" \
	--checkpoints_total_limit="1" \
	--validation_steps="50" \
	--tracker_run_name="test" \
	--tracker_project_name="sptnr-sdxl-lvchengguiyuxi" \
	--validation_guidance="7" \
	--validation_guidance_rescale="0" \
	--use_8bit_adam \
	--use_ema \
--enable_xformers_memory_efficient_attention \
--set_grads_to_none \
--freeze_encoder \
--gradient_checkpointing

**Comment by bghira** [comment]
I've taken a whack at it, if you can figure out a solution, I'll include it.

**Comment by zhuliyi0** [comment]
this works:

`        accelerator.init_trackers(
            args.tracker_project_name,
            config=vars(public_args),
            init_kwargs={
                "wandb":
                {
                    "name": args.tracker_run_name,
                    "id": f"{args.tracker_project_name},{args.tracker_run_name}",
                    "resume": "allow",
                }
            },
        )
`

**Comment by bghira** [comment]
thanks
